src/mavon-editor.vue

```diff
@@ -374,7 +374,11 @@ export default {
                 'js': loadScript
             };
             if (_obj.hasOwnProperty(type)) {
-                _obj[type](this.p_external_link[name](), callback);
+                let id = null
+                if(name === 'markdown_css'){
+                    id = 'md-code-style'
+                }
+                _obj[type](this.p_external_link[name](), callback, id);
```

Comment:
There is no point in adding the id of `md-code-style` to the `link` node of `markdown_css` because `markdown_css` is fixed and will not switch to other style sheets.
https://github.com/hinesboy/mavonEditor/blob/88585f2b221cb1c0d902da7eb0666fcb96fa21e8/src/mavon-editor.vue#L284-L286


---
给`markdown_css`的`link`节点增加`md-code-style` id没有意义，因为`markdown_css`是固定的不会切换成其他样式表